tests/name_constraints.rs

```diff
@@ -56,7 +56,7 @@ fn check_cert(
     Ok(())
 }
 
-// DO NOT EDIT BELOW: generated by name_constraints/generate.py
+// DO NOT EDIT BELOW: generated by tests/generate.py
```

Comment:
I think this should be in the same commit where the new boundary is used?

Reply:
Moved into the re-generation commit.